partiql-planner/src/main/kotlin/org/partiql/planner/internal/IdentifierManager.kt

```diff
@@ -0,0 +1,37 @@
+package org.partiql.planner.internal
+
+import org.partiql.ast.Binder
+import org.partiql.ast.Identifier
+import org.partiql.ast.binder
+import org.partiql.ast.identifierSymbol
+
+// The identifier manager class takes in two parameters,
+// in case we want to decouple case preservation and case normalization behavior in the future.
```

Comment:
We have explicitly decided NOT to decouple these per our latest discussion. This is an internal class, so we can always change this later, but for now let's not create edge-cases that we've explicitly decided against.